corehq/util/context_processors.py

```diff
@@ -175,6 +175,27 @@ def commcare_hq_names(request=None):
     }
 
 
+def server_location_display(request):
+    SERVER_LOCATION_DISPLAY = {
+        'production': {
+            'flag': "🇺🇸",
+            'location_code': "US",
+        },
+        'india': {
+            'flag': "🇮🇳",
+            'location_code': "IN",
```

Comment:
Is there a specific reason to use location codes as opposed to names? Mainly thinking that  "India Server" could be better than "IN Server". The other two would likely stay the same.

Similarly, could also translate these names? Mainly if we do switch to India instead of IN, translating that seems ideal. But maybe there is benefit to translating US and EU as well.

Reply:
Yeah, I do think "India Server" reads nicer than "IN", especially given US and EU are common English abbreviations and IN is not. I believe this was feedback from @kcowger that if we're using 2-letter codes in this particular display, we should use them consistently.

On translation, I think if we're using two-letter codes it probably isn't necessary (and it wouldn't be able to be translated in context, would it? so a translator would see "IN" without knowing that's meant to be the country code for India.) But if we use "India" it should definitely be translated. `gettext_lazy` would be the appropriate way to do that here, right?

Reply:
Actually while we are at it, since it is already under the context of "Server Location", I think we can just say "India", "European Union" and "United States" right? Or at least for the settings dropdown.